CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
+## v0.5.4-alpha
+* Add Snowflake role in database links
```

Comment:
`Add Snowflake role in database link definition`?

The role was always involved, we are now storing it, which is an internal detail that happens to lead to the role also being returned when describing a database link